### PR TITLE
fantom - add MIM on fantom to token price list [WIP]

### PR DESCRIPTION
**Pull request by RantumBits**
add MIM to token price list

### CODE REVIEW

**Comment by RantumBits** [comment]
not clear why - token is here - https://coinpaprika.com/coin/mim-magic-internet-money/

**Comment by RantumBits** [comment]
This failed the price automation test, due to formatting for DuneSQL. 

When I tried updating only legacy models, this failed due to Dune SQL model missing 'dunesql' tag.

 Can you advise @Hosuke ?

**Comment by Hosuke** [comment]
> This failed the price automation test, due to formatting for DuneSQL.
> 
> When I tried updating only legacy models, this failed due to Dune SQL model missing 'dunesql' tag.
> 
> Can you advise @Hosuke ?

We may merge with the main first.

File: models/prices/fantom/prices_fantom_tokens.sql [hunk 1]
```diff
@@ -71,5 +71,6 @@ FROM
     ('2shares-2share', 'fantom', '2SHARES', 0xc54A1684fD1bef1f077a336E6be4Bd9a3096a6Ca, 18), 
     ('3share-3share', 'fantom', '3SHARE', 0x6437adac543583c4b31bf0323a0870430f5cc2e7, 18),
     ('ico-axelar', 'fantom', 'AXL', 0x8b1f4432f943c465a973fedc6d7aa50fc96f1f65, 6),   
-    ('grain-granary', 'fantom', 'GRAIN', 0x02838746d9e1413e07ee064fcbada57055417f21, 18)
+    ('grain-granary', 'fantom', 'GRAIN', 0x02838746d9e1413e07ee064fcbada57055417f21, 18),
+    ('mim-magic-internet-money', 'fantom', 'MIM', 0x82f0b8b456c1a451378467398982d4834b6829c1, 18)
```
Comment (Hosuke) [inline models/prices/fantom/prices_fantom_tokens.sql:75]: https://ftmscan.com/token/0x82f0b8b456c1a451378467398982d4834b6829c1

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits

File: models/prices/fantom/prices_fantom_tokens.sql (hunk 1 above)
Comment (Hosuke) [reply models/prices/fantom/prices_fantom_tokens.sql:75]: API: https://api.coinpaprika.com/v1/coins/mim-magic-internet-money

**Review by jeff-dude — APPROVED** [review]
thank you both!

**Pull request merged**